### ISSUE TITLE
🐛 Bug Report: UX/UI problem with the password field

### ISSUE THREAD

**Issue by kergian** [appwrite/appwrite#5790]
### 👟 Reproduction steps

UX/UI problem with the password field.
If client uses some kind of password manager, password manager adds its UI element to password input fields, to auto-swap or generate new password or something like that.
Appwrite password showing control is overlapped with password manager control (see screenshot)

As a variant of proper UX/UI practice, you can consider adding an additional checkbox that, when activated, will display the entered password (see screenshot)

### 👍 Expected behavior

I drew an additional checkbox `Show password`, which is placed outside the password input field. And, as you can see, it is not interfere with the control added by the password manager:

<img width="508" alt="Screenshot 2023-07-09 at 02 03 20" src="https://github.com/appwrite/appwrite/assets/134743498/403b2561-1c27-44b3-8c5f-fbe242336080">

### 👎 Actual Behavior

Here, as you can see, the password mapping control added by Appwrite overlaps with the control added by the password manager:

<img width="516" alt="Screenshot 2023-07-09 at 01 53 43" src="https://github.com/appwrite/appwrite/assets/134743498/92ecd4b7-0d7d-429e-8a44-153b412d3fd8">
<img width="510" alt="Screenshot 2023-07-09 at 01 54 07" src="https://github.com/appwrite/appwrite/assets/134743498/5cfa0d4b-b684-48c9-8b22-f8aa142e8406">

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by kergian** [comment]
On `Actual behavior` watch how the password manager controls overlap with the Appwrite `show password` control

**Comment by joeyouss** [comment]
Hi
I tried it, and seems like this is browser specific - but we are always welcoming suggestions for enhancing and customizing the user experience across different environments. 

Thank you for opening this, and we will surely consider this in the near future.

**Comment by safwanyp** [comment]
This was discussed in one of the recent office hours, and it's a very-browser specific issue as @joeyouss mentioned. 

The only way (i can think of) to get rid of this issue would be to change the `input` type from `password` to `text`. This will get rid of the auto-fill as password managers (3rd party and browser) look for the `<input type="password">` element. 

Changing the type to `text` will work, but any password entered will be unmasked by default. But that should be a simple fix with some JS. 

If this is an issue that Appwrite is open to fixing, I'd be happy to create a PR with a fix on the console repo!

**Comment by kergian** [comment]
@safwanyp why not just create a checkbox when clicked that shows the password in the input (like on my screenshot)? This is a fairly common practice in UI design, and it will solve this and potentially any future problems if some software adds its own control to the password field.

**Comment by joeyouss** [comment]
Hello @kergian , after careful consideration, we have determined that the components do not overlap and remain clickable. Therefore, we have decided to keep them as they are for now. While we acknowledge that adopting the checkbox may be a possibility in the future, our current focus is on other pressing tasks, such as the upcoming release. Thank you for understanding. I will be closing this issue since it is not being worked on as of now but it can surely be open in future for fixes.
